README.md

```diff
@@ -56,6 +72,7 @@ Describe the importance of using object-oriented programming.
 Your Answer:
 ```js
 // your answer here
+object oriented programming makes the code neat and dry.
```

Comment:
OOP does help to organize code.  The result usually is neat and can be DRY.  I recommend you mention encapsulation (with description) and reflect real world objects in code.